### PR TITLE
Gitpod Self-Hosted Release 0.4.0 announcement

### PR DESCRIPTION
**Pull request by meysholdt**

### CODE REVIEW

File: src/blog/gitpod-self-hosted-0.4.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Gitpod Self-Hosted is truly self-hosted: it does not require an internet connection to operate as every single bit runs on your network, and Gitpod Self-Hosted won't call home.
+
+Operating Gitpod in production on gitpod.io, has enabled us to battle-harden this product at scale. Both offerings share the same codebase, hence offer the same qualities regarding scalability, cloud-density, reliability, and security. All improvements that we make on Gitpod’s SaaS offering directly benefit it’s self-hosted brethren.
+
+This release ships a plenthora of bugfixes and improvements, particularly in the area of GitLab integration. 
+There is one major feature that I want to show in more detail since you can't try this on gitpod.io:
+
+## New Admin UI
+
+Besides a plenthora of bug fixes, this release contains
```
Comment (svenefftinge) [inline src/blog/gitpod-self-hosted-0.4.0.md:24]: plethora

Comment (svenefftinge) [reply src/blog/gitpod-self-hosted-0.4.0.md]: Maybe:

> In addition to all the bugfixes and enhancements from Gitpod.io, the release contains

File: src/blog/gitpod-self-hosted-0.4.0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+This release ships a plenthora of bugfixes and improvements, particularly in the area of GitLab integration. 
+There is one major feature that I want to show in more detail since you can't try this on gitpod.io:
+
+## New Admin UI
+
+Besides a plenthora of bug fixes, this release contains 
+
+An Admin UI: There is a new web UI for administrative tasks available at `mygitpod.tld/admin`. The UI is handy when providing support for users: It allows us to list users and workspaces and to inspect their details.
+
+![Homebrew in Gitpod screenshot](./self-hosted-0.4.0/sh-userlist2.png)
```
Comment (svenefftinge) [inline src/blog/gitpod-self-hosted-0.4.0.md]: Remove the first screenshot?

File: src/blog/gitpod-self-hosted-0.4.0.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+author: meysholdt
+subtitle: Features, Fixes, Fixes, and Fixes
+title: Gitpod Self-Hosted 0.4.0
+image: /globe.png
+teaserImage: /globe.png
+---
+We are excited to announce the new release of Gitpod Self-Hosted in version 0.4.0. Gitpod Self-Hosted allows everyone to set-up and run  a service with the same features as gitpod.io in your cloud or corporate network. 
+
+Gitpod Self-Hosted runs on Kubernetes and is easily installed via "helm install". On gitpod.io you can find detailed documentation and some automation to install Gitpod on vanilla Kubernetes or on Google Cloud Platform. For the next release, we plan to add support for AWS, Azure, and OpenShift.
+
```
Comment (svenefftinge) [inline src/blog/gitpod-self-hosted-0.4.0.md:12]: I'm missing a headline here, to make the text more digestable.
Maybe
```
### How Does it Work?
```

File: src/blog/gitpod-self-hosted-0.4.0.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Users log into Gitpod via OAuth from your self managed Gitlab or GitHub enterprise. This ensures that all access privileges from your Git hosting solution are taken into account and that there is no additional administrative effort for managing user accounts.
+
+Gitpod Self-Hosted is truly self-hosted: it does not require an internet connection to operate as every single bit runs on your network, and Gitpod Self-Hosted won't call home.
+
+Operating Gitpod in production on gitpod.io, has enabled us to battle-harden this product at scale. Both offerings share the same codebase, hence offer the same qualities regarding scalability, cloud-density, reliability, and security. All improvements that we make on Gitpod’s SaaS offering directly benefit it’s self-hosted brethren.
+
+This release ships a plenthora of bugfixes and improvements, particularly in the area of GitLab integration. 
+There is one major feature that I want to show in more detail since you can't try this on gitpod.io:
+
+## New Admin UI
```
Comment (svenefftinge) [inline src/blog/gitpod-self-hosted-0.4.0.md:22]: I'd use h3 (i.e. `### New Admin UI`)

File: src/blog/gitpod-self-hosted-0.4.0.md (hunk 1 above)
Comment (meysholdt) [reply src/blog/gitpod-self-hosted-0.4.0.md]: fixed.

File: src/blog/gitpod-self-hosted-0.4.0.md (hunk 4 above)
Comment (meysholdt) [reply src/blog/gitpod-self-hosted-0.4.0.md]: fixed.

File: src/blog/gitpod-self-hosted-0.4.0.md (hunk 3 above)
Comment (meysholdt) [reply src/blog/gitpod-self-hosted-0.4.0.md:14]: fixed.

File: src/blog/gitpod-self-hosted-0.4.0.md (hunk 2 above)
Comment (meysholdt) [reply src/blog/gitpod-self-hosted-0.4.0.md]: fixed.

File: src/blog/gitpod-self-hosted-0.4.0.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Users log into Gitpod via OAuth from your self managed Gitlab or GitHub enterprise. This ensures that all access privileges from your Git hosting solution are taken into account and that there is no additional administrative effort for managing user accounts.
+
+Gitpod Self-Hosted is truly self-hosted: it does not require an internet connection to operate as every single bit runs on your network, and Gitpod Self-Hosted won't call home.
+
+Operating Gitpod in production on gitpod.io, has enabled us to battle-harden this product at scale. Both offerings share the same codebase, hence offer the same qualities regarding scalability, cloud-density, reliability, and security. All improvements that we make on Gitpod’s SaaS offering directly benefit it’s self-hosted brethren.
```
Comment (geropl) [inline src/blog/gitpod-self-hosted-0.4.0.md:19]: Superfluous `,`

Comment (geropl) [reply src/blog/gitpod-self-hosted-0.4.0.md:19]: I'm by far not native speaker but `brethren` sounds weirdly archaic and reminds me of Lord of the Rings. Maybe: `... directly benefit the self-hosted version.`

File: src/blog/gitpod-self-hosted-0.4.0.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+![Gitpod Admin UI](./self-hosted-0.4.0/sh-userdetails2.png)
+
+To try the new admin interface in Gitpod Self-Hosted, you'll need to assign the "admin" privilege to your user and open /admin in your browser.
+
+### Getting Started
+
+The fastest way to get started is by installing Gitpod Self-Hosted [on Google Cloud Platform](https://www.gitpod.io/docs/self-hosted/0.4.0/install/install-on-gcp-script/). There is also more in-depth documentation available on how to install Gitpod on [vanilla Kubernetes](https://www.gitpod.io/docs/self-hosted/0.4.0/install/install-on-kubernetes/).
+
+Gitpod Self-Hosted is free for up to five users. For more users, [please click here to start a trial](https://gitpod.io/selfhosted-trial/) or see our [pricing page](https://www.gitpod.io/pricing/#self-hosted).
```
Comment (svenefftinge) [inline src/blog/gitpod-self-hosted-0.4.0.md:38]: `please click here` sounds fishy.
How about:
```
For more users, you can obtain a free one month trial [here](...) ...
```

**Pull request merged**